features/ultils/world.ts

```diff
@@ -17,6 +19,18 @@ export class OurWorld extends World {
   public set response(value) {
     this._response = value;
   }
+
+  public init(options: { rootDir: string }) {
+    this.workingDir = options.rootDir;
+    this.log(`Root working directory: ${this.workingDir}`);
+  }
+
+  public initForIsolatedScenario() {
+    this.workingDir = fs.mkdtempSync(
+      this.workingDir ? path.join(this.workingDir, "case-") : "case-",
+    );
+    this.log(`Scenario working directory: ${this.workingDir}`);
+  }
```

Comment:
I think the `init()` should know only the `workingDir` where the World should work at.

I mean, the processes in `initForIsolatedScenario` should be performed in `Before({ tags: "@isolated" }`

Reply:
@mshrtsr 

Could you explain more details about this?
Now, we have two Before hooks.

```
// Because World isn't available in BeforeAll, we have to use Before hook.
Before(function () {
  this.init({ rootDir });
});

Before({ tags: "@isolated" }, function () {
  this.initForIsolatedScenario();
});
```

Reply:
I mean the following.
Only the `Before()` hook know about the `isolated`.

## In hooks.ts

```typescript
BeforeAll(function () {
  rootDir = fs.mkdtempSync(
    path.join(os.tmpdir(), `cli-kintone-e2e-test-${new Date().valueOf()}-`),
  );
  console.log(`Root working directory: ${rootDir}`)
});

Before(function () {
  this.init({ workingDir: rootDir });
});

Before({ tags: "@isolated" }, function () {
  const workingDir = fs.mkdtempSync(path.join(rootDir, "case-") : "case-");
  this.init({ workingDir });
});
```

## In world.ts

```typescript
export class OurWorld extends World {
  public workingDir?: string;

  ...

  public init(options: { workingDir: string }) {
    this.workingDir = options.workingDir;
    this.log(`Scenario working directory: ${this.workingDir}`);
  }

  // Remove initForIsolatedScenario()
}
```

Reply:
@hung-cybo 
I mean the following.
Only the `Before()` hook know about the `isolated`.

## In hooks.ts

```typescript
BeforeAll(function () {
  rootDir = fs.mkdtempSync(
    path.join(os.tmpdir(), `cli-kintone-e2e-test-${new Date().valueOf()}-`),
  );
  console.log(`Root working directory: ${rootDir}`)
});

Before(function () {
  this.init({ workingDir: rootDir });
});

Before({ tags: "@isolated" }, function () {
  const workingDir = fs.mkdtempSync(path.join(rootDir, "case-") : "case-");
  this.init({ workingDir });
});
```

## In world.ts

```typescript
export class OurWorld extends World {
  public workingDir?: string;

  ...

  public init(options: { workingDir: string }) {
    this.workingDir = options.workingDir;
    this.log(`Scenario working directory: ${this.workingDir}`);
  }

  // Remove initForIsolatedScenario()
}
```